cfgov/v1/jinja2tags/__init__.py

```diff
@@ -66,25 +63,6 @@ def is_filter_selected(context, fieldname, value):
     return value in query_string_values
 
 
-def render_stream_child(context, stream_child):
-    rendered = stream_child.render(context=context)
-
-    # This logic is needed because historically we have supported the
-    # inclusion of raw HTML tags in any Wagtail text or rich text block.
-    # Ideally we could remove this logic, but before we do so we need to
-    # eliminate all such tags from our field content.
-    #
-    # By default all blocks are unescaped, but individual blocks can disable
-    # this behavior by setting unescape=False in their Meta class. Once all
```

Comment:
With this change, can we remove all places where block Metas set unescape, [for example](https://github.com/cfpb/consumerfinance.gov/blob/3c972e1b783bffeea2ca5d21abfe8b5f39b8f91f/cfgov/v1/atomic_elements/tables.py#L26)?

Reply:
Done in [356226a](https://github.com/cfpb/consumerfinance.gov/pull/8169/commits/356226a486e5307042035cb7acec31ff35d243d0)!